_posts/2023-05-06-reverse-engineering-gba.markdown

```diff
@@ -0,0 +1,1062 @@
+---
+layout: post
+title: Reverse Engineering a GBA Game
+date: 2023-05-06 05:00:00
+img: reverse-engineering/debug-env.png
+description: "How to reverse engineer a GBA game, using Pokémon Mystery Dungeon: Red Rescue Team as an example."
```

Comment:
```suggestion
description: "How to reverse engineer a GBA game using "Pokémon Mystery Dungeon: Red Rescue Team" as an example."
```

Reply:
Good call, though I've seen italics more than quotes so I'll do that.